### ISSUE TITLE
Verified accounts

### ISSUE THREAD

**Issue by edsu** [DocNow/docnow#45]
It could be useful to display verified users in explore mode, or perhaps after the collection has started. I think highlighting the verified users could be useful because they are individuals who have gone the extra step to authenticate their account, and could be more interested in their content having a home in an archive.

**Comment by alexandradm** [comment]
definitely. can we just pull in their verified symbol, or should i find a font awesome icon type thing and have that display if the metadata on verified comes in positive? i would say we put it next to user handle as its done on twitter.
